src/main/java/org/springframework/integration/aws/outbound/KplMessageHandler.java

```diff
@@ -63,11 +64,16 @@
 import org.springframework.util.StringUtils;
 
 /**
- * The {@link AbstractMessageHandler} implementation for the Amazon Kinesis Producer
- * Library {@code putRecord(s)}.
+ * The {@link AbstractMessageHandler} implementation for the Amazon Kinesis Producer Library {@code putRecord(s)}.
+ * <p>
+ * {@link KplBackpressureException} is thrown when backpressure handling is enabled and buffer is at max capacity.
+ * The exception can be handled with
```

Comment:
`This exception`?
I mean sounds better and have a connection with the previous sentence.